src/hdr-config.cpp

```diff
@@ -242,25 +242,28 @@ namespace librealsense
         {
             if (validate_config())
             {
-                // saving status of options that are not compatible with hdr,
-                // so that they could be reenabled after hdr disable
-                set_options_to_be_restored_after_disable();
-
-                if (_use_workaround)
+                if (!_is_enabled)
```

Comment:
When a 3rd party disables HDR in-between the two calls this cached value may not reflect the actual status. Can it be retrieved from FW just before this line?

Reply:
done